Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorWindow.cs

```diff
@@ -356,8 +357,8 @@ private bool TryUpdateFromAsset()
                 var asset = AssetDatabase.LoadAssetAtPath<InputActionAsset>(assetPath);
                 workingCopy = InputActionAssetManager.CreateWorkingCopy(asset);
                 m_AssetJson = InputActionsEditorWindowUtils.ToJsonWithoutName(asset);
-                m_IsDirty = false;
                 m_State = new InputActionsEditorState(m_State, new SerializedObject(workingCopy));
```

Comment:
Updated ordered here to be consistent with the new/added code since m_IsDirty is relying on serializedObject from the state its an incorrect order to set m_IsDirty afterwards.

Reply:
In a post 1.8.0 refactor I would suggest getting rid of m_IsDirty completely since it can always be computed, and delay its computation until its actually needed instead.